english/candyland/rainbow/rainbow.md

```diff
@@ -15,6 +15,6 @@ Yellow taste like barf, and green taste like boogers.
 
 You remember your past and feel ashamed you know what those taste like. 
 
-You begin to cry uncontrollably.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.